### ISSUE TITLE
🐛 [BUG] - Font and Font Size Not Applied in View Mode

### ISSUE THREAD

**Issue by Abdulrehmanharisdev** [walkframe/gridsheet#58]
### Description

I am dynamically applying styles to each cell in my GridSheet component, including font, fontSize, and other properties. These styles are correctly applied and displayed when the cells are  being edited. However, when I focus out and in view mode, the font and fontSize properties are not retained. Instead, the default font and font size are shown during.
![image](https://github.com/user-attachments/assets/008d6c12-4366-4702-8a5a-edc8f00822af)

### Reproduction URL

https://docs.walkframe.com/products/gridsheet/react/

### Reproduction steps

```bash
I am dynamically applying styles to each cell in my GridSheet component, including font, fontSize, and other properties. These styles are correctly applied and displayed when the cells are  being edited. However, when I focus out and in view mode, the font and fontSize properties are not retained. Instead, the default font and font size are shown during.
```

### Browsers

Chrome

### OS

Windows

**Comment by righ** [comment]
Thanks for reporting the bug.
Please check back as I have fixed it in v1.0.7.

**Comment by Abdulrehmanharisdev** [comment]
when it will be merged ?

**Comment by righ** [comment]
I pushed a fix commit da10ca1 to master, so no need to merge.
The font size has been fixed at https://www.npmjs.com/package/@gridsheet/react-core/v/1.0.7 .
The other elements have been reported by you in #60, so please close this Issue when you are sure that the font size is fixed in this version.

**Comment by Abdulrehmanharisdev** [comment]
Thanks it fixed!
